src/test/resources/no/api/freemarker/java8/time/datetime.feature

```diff
@@ -33,49 +32,92 @@ Feature: Test the date time functionality
         And Instant object for "2007-12-03T10:15:30.00Z"
         Then expect the template to return "2007-12-03T10:15:30Z"
 
-
     ### LocalDate ###
     Scenario: Test basic LocalDate use in template
         Given a template "${obj}"
         And LocalDate object for "2007-12-03"
         Then expect the template to return "2007-12-03"
 
+    Scenario: Test LocalDate use in template with custom pattern
+        Given a template "${obj.format('EEEE d MMMM yyyy')}"
+        And LocalDate object for "2007-12-03"
+        Then expect the template to return "mandag 3 desember 2007"
```

Comment:
Yes, since that is set in top of the features-file. But good point. I will move it to all the scenarios to make it easier to understand.